.circleci/config.yml

```
@@ -68,7 +69,7 @@ jobs:
           name: Install dependencies
           command: |
             pip install -e .
-            pip install nbconvert ipykernel
+            pip install nbconvert ipykernel tqdm ipywidgets
```

```suggestion
            pip install nbconvert ipykernel
```

I don't think this job needs `tqdm ipywidgets`.